benchbuild/utils/actions.py

```diff
@@ -126,10 +133,10 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
-def log_before_after(name: str, desc: str):
+def log_before_after(name: str, desc: str) -> Callable:
     """Log customized stirng before & after running func."""
 
-    def func_decorator(f):
+    def func_decorator(f) -> Callable:
```

Comment:
C0103 (invalid-name): Argument name "f" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern  <!-- Automatically posted by Lintly -->